scripts/tests_and_coverage.sh

```diff
@@ -25,7 +25,8 @@ set +u
 
 if [ -z "${GITHUB_ACTIONS}" ] || [ "$1" == "test_and_coverage" ] ; then
     # Python tests (rerun with debug if failures)
-    PYTHONPATH=$(pwd) pytest -qx tests/ --cov='.' --no-cov-on-fail --cov-report term-missing || PYTHONPATH=$(pwd) pytest -xrpP --log-level debug tests/
+    # PYTHONPATH=$(pwd) pytest -qx tests/ --cov='.' --no-cov-on-fail --cov-report term-missing || 
+    PYTHONPATH=$(pwd) pytest -xrpP --log-level debug tests/
```

Comment:
Any specific reason to change this? The reasoning behind it was only do limited output tests unless (`||`) something fails. If we do it this way you always get excessive output and no more coverage details?

Reply:
I do this to be able to see the full test-output while I'm trying out new code. Will return to original when I think the changes are beneficial.

Note: this PR is in draft-status, I wanted to see the Codecov-report to get an idea about the impact of the changes.